js/test/TopNav.js

```diff
@@ -727,11 +727,11 @@ describe('TopNav', function () {
             cleanup();
         });
 
+        // TODO triggers error messages in jsDOM version 12, but passes
```

Comment:
```suggestion
        // TODO triggers error messages in jsDOM since version 12, but passes
```

Or do you mean this? (Otherwise I again don't get the version)

Also this is hardly a TODO, is it, or what may be done about it? Maybe change it to a `NOTE` (or just a usual comment) and if you consider this behavior a bug, then we shall report it to JSDOM.

Leaving TODOs in code is usually not good otherwise :wink: